indico/modules/events/contributions/templates/display/contribution_display.html

```diff
@@ -4,6 +4,7 @@
 {% from 'events/display/indico/_common.html' import render_location, render_users, render_speakers %}
 {% from 'events/papers/_contributions.html' import render_paper_section, render_editables_section, render_proceedings_section %}
 {% from 'message_box.html' import message_box %}
+{% from 'events/timetable/display/indico/_common.html' import render_references %}
```

Comment:
Don't you want to move this one up one line? It looks much more nicely sorted that way :P